### PR TITLE
snowflake-sqlalchemy 1.6.1 :snowflake:

### PR DESCRIPTION
**Pull request by tombenes**
snowflake-sqlalchemy 1.6.1 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5932]
- dev_url:        https://github.com/snowflakedb/snowflake-sqlalchemy/tree/v1.6.1
- conda_forge:    https://github.com/conda-forge/snowflake-sqlalchemy-feedstock
- pypi:           https://pypi.org/project/snowflake-sqlalchemy/1.6.1/
- pypi inspector: https://inspector.pypi.io/project/snowflake-sqlalchemy/1.6.1/

### Explanation of changes:

- new version number

[PKG-5932]: https://anaconda.atlassian.net/browse/PKG-5932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,13 +19,12 @@ requirements:
   host:
```
Comment (msentissi) [inline recipe/meta.yaml:19]: I see `requires-python = ">=3.8"`, so we should skip # [py<**38**].

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,13 +19,12 @@ requirements:
...
     - pip
     - python
-    - setuptools
-    - wheel
+    - hatchling
   run:
     - python
     - importlib-metadata  # [py<38]
```
Comment (msentissi) [inline recipe/meta.yaml]: I don't see `importlib-metadata` in the .toml file. Is it needed ?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -19,13 +19,12 @@ requirements:
...
     - snowflake-connector-python <4.0.0
-    - sqlalchemy <2.0.0,>=1.4.0
+    - sqlalchemy >=1.4.19
 
 test:
```
Comment (msentissi) [inline recipe/meta.yaml:28]: Is it possible to run some of the tests under `tests/` ?

Comment (msentissi) [reply recipe/meta.yaml:28]: For `doc_url`, would this be a better alternative : https://docs.snowflake.com/en/developer-guide/python-connector/sqlalchemy ?

Comment (skupr-anaconda) [reply recipe/meta.yaml:28]: Yes, but without a language variant is even better https://docs.snowflake.com/developer-guide/python-connector/sqlalchemy

File: recipe/meta.yaml (hunk 1 above)
Comment (tombenes) [reply recipe/meta.yaml:19]: Done

File: recipe/meta.yaml (hunk 2 above)
Comment (tombenes) [reply recipe/meta.yaml]: As we don't build for py<38 , I removed it

File: recipe/meta.yaml (hunk 3 above)
Comment (tombenes) [reply recipe/meta.yaml:28]: I've updated doc_url

Comment (tombenes) [reply recipe/meta.yaml:28]: Tests require connecting to snowflake, I don't think we should do that.

Comment (msentissi) [reply recipe/meta.yaml:28]: If all the tests require connecting to snowflake, I agree we should skip them

**Review by msentissi — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**